cmd/celestia/main.go

```diff
@@ -36,12 +29,17 @@ func main() {
 }
 
 func run() error {
+	cfg := sdk.GetConfig()
+	cfg.SetBech32PrefixForAccount(app.Bech32PrefixAccAddr, app.Bech32PrefixAccPub)
+	cfg.Seal()
+
```

Comment:
Why not init? 
Also, It seems weird that this is required for `celestia` binary to function. Can we also extract it into the separate bin or it is required for stateaccess to function?

Reply:
good catch